components/script_injector/renderer/BUILD.gn

```diff
@@ -20,3 +20,14 @@ component("renderer") {
     "//third_party/blink/public:blink",
   ]
 }
+
+source_set("unit_tests") {
+  testonly = true
+  sources = [ "script_injector_unittest.cc" ]
+  deps = [
+    "//base/test:test_support",
+    "//brave/components/script_injector/renderer:renderer",
+    "//components/prefs:test_support",
+  ]
+  defines = [ "HAS_OUT_OF_PROC_TEST_RUNNER" ]
```

Comment:
this is only needed for browser tests https://source.chromium.org/chromium/chromium/src/+/main:content/public/test/browser_test.h;l=49?q=browser_test.h&ss=chromium